velocity/src/main/java/me/confuser/banmanager/velocity/Cancellable.java

```diff
@@ -0,0 +1,22 @@
+package me.confuser.banmanager.velocity;
+// This totally isn't stolen from https://github.com/SpigotMC/BungeeCord/blob/master/api/src/main/java/net/md_5/bungee/api/plugin/Cancellable.java
```

Comment:
Be cautious of Bungee's licence terms, this current copy/paste is in violation of it

Reply:
I have no idea how to resolve this issue, any ideas?